README.md

```diff
@@ -103,7 +103,7 @@ jupyter lab
 
 JupyterLab will open automatically in your browser. Now, you can run the
 notebook
-[run_model_in_notebook.ipynb](https://github.com/EcoExtreML/STEMMUS_SCOPE_Processing/blob/main/notebooks/run_model_in_notebook.ipynb).
+[run_model_in_notebook.ipynb](https://github.com/EcoExtreML/STEMMUS_SCOPE_Processing/blob/main/docs/notebooks/run_model_in_notebook.ipynb).
```

Comment:
The link will show up in "Getting started" in the doc. It should point to a tag instead of main.

Reply:
I removed duplicate info from the README.md file.